sktime/forecasting/statsmodels_arima.py

```diff
@@ -0,0 +1,316 @@
+            method=self.method,
+            method_kwargs=self.method_kwargs,
+            gls=self.gls,
+            gls_kwargs=self.gls_kwargs,
+            cov_type=self.cov_type,
+            cov_kwds=self.cov_kwds,
+            return_params=self.return_params,
+            low_memory=self.low_memory,
+        )
+
+    def summary(self):
+        """Get a summary of the fitted forecaster.
+
+        This is the same as the implementation in statsmodels:
+        https://www.statsmodels.org/dev/examples/notebooks/generated/statespace_structural_harvey_jaeger.html
+        """
+        return self._fitted_forecaster.summary()
+
+    @staticmethod
+    def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
+        """Construct confidence interval at specified `alpha` for each timestep.
+
+        Parameters
+        ----------
+        prediction_results : PredictionResults
+            results class, as returned by ``self._fitted_forecaster.get_prediction``
+        alpha : float
+            one minus nominal coverage
+
+        Returns
+        -------
+        pd.DataFrame
+            confidence intervals at each timestep
+
+            The dataframe must have at least two columns ``lower`` and ``upper``, and
+            the row indices must be integers relative to ``self.cutoff``. Order of
+            columns do not matter, and row indices must be a superset of relative
+            integer horizon of ``fh``.
+        """
+        conf_int = prediction_results.conf_int(alpha=alpha)
+        conf_int.columns = ["lower", "upper"]
+
+        return conf_int
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : list of dict, default = []
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        return [
+            {
+                "order": (0, 1, 2),
+                "trend": "t",
+                "enforce_stationarity": False,
+                "enforce_invertibility": False,
+                "concentrate_scale": True,
+                "method": "statespace",
+            },
+            {
+                "order": (
+                    1,
+                    1,
+                    2,
+                ),
+                "trend": "t",
```

Comment:
May be one test case can be with no trend, i.e. `n`, case as well? Linear trend is covered above already.

Reply:
Let me try